File: app/src/java/java/com/bear2b/sampleapp/ui/view/activities/MainActivity.java
package com.bear2b.sampleapp.ui.view.activities;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.bear2b.sampleapp.R;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.basic_sample).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, BasicSampleActivity.class));
            }
        });
        findViewById(R.id.advanced_sample).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, AdvancedSampleActivity.class));
            }
        });
    }
}

File: app/src/kotlin/java/com/bear2b/sampleapp/ui/view/activities/MainActivity.kt
package com.bear2b.sampleapp.ui.view.activities

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View
import com.bear2b.sampleapp.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        basic_sample.setOnClickListener { startActivity(Intent(this, BasicSampleActivity::class.java)) }

        advanced_sample.setOnClickListener { startActivity(Intent(this, AdvancedSampleActivity::class.java)) }
    }
}